tools/confix/go.mod

```diff
@@ -0,0 +1,3 @@
+module cosmossdk.io/confix
```

Comment:
I don't think this needs to be separate? I think it running on startup of a node is all that's needed?

Reply:
It is to make part two of reusable for older chains without backporting. A bitnlike Rosetta; where you simply need to add the command. Or use it standalone.